lib/generators/administrate/dashboard/dashboard_generator.rb

```diff
@@ -71,6 +71,21 @@ def field_type(attribute)
         end
       end
 
+      def column_type_for_attribute(attr)
+        type = klass.column_types[attr].type
+
+        if type == :integer && enum_type?(attr)
+          :string
```

Comment:
How about changing this to `:enum`, so we can treat enum fields and string fields a bit differently?